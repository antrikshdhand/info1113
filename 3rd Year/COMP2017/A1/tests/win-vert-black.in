place A1
place J1
place A2
place G2
place A3
place N1
place A4
place C1
place A5
