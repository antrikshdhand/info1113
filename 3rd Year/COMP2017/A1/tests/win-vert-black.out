Black wins!
A1J1A2G2A3N1A4C1A5
Thank you for playing!
